vg/vgimg/vgimg.go

```diff
@@ -164,6 +164,14 @@ func UseImageWithContext(img draw.Image, gc draw2d.GraphicContext) option {
 	}
 }
 
+// Image returns the image the canvas is drawing to.
+//
+// Users are not supposed to directly modify the image once the canvas
```

Comment:
yes, the size (ie: via `draw.Image.Bounds()` and then `image.Rectange{ Min, Max Point }`).